core/templates/components/state-directives/solution-editor/solution-explanation-editor.component.html

```diff
@@ -1,49 +1,50 @@
 <div class="h-100">
-  <div ng-if="!$ctrl.explanationEditorIsOpen"
+  <div *ngIf="!explanationEditorIsOpen"
        class="h-100"
-       ng-attr-title="<[$ctrl.isEditable ? 'Edit explanation' : '']>">
+       [attr.title]="isEditable ? 'Edit explanation' : ''">
     <div class="oppia-readonly-rule-tile"
-         ng-class="{'oppia-editable-section': $ctrl.isEditable}">
+         [ngClass]="{'oppia-editable-section': isEditable}">
       <div class="oppia-rule-preview oppia-transition-200">
         <div class="oppia-click-to-start-editing"
-             ng-if="$ctrl.isEditable" ng-click="$ctrl.openExplanationEditor()">
+             *ngIf="isEditable" (click)="openExplanationEditor()">
           <i class="material-icons oppia-editor-edit-icon float-right"
              title="Edit Explanation">&#xE254;
           </i>
         </div>
         <strong>Explanation:</strong>
         <span>
-          <angular-html-bind class="oppia-rte-editor" html-data="$ctrl.StateSolutionService.savedMemento.explanation.html">
-          </angular-html-bind>
+          <angular-html-bind-wrapper classStr="oppia-rte-editor" [htmlData]="stateSolutionService.savedMemento.explanation.html">
+          </angular-html-bind-wrapper>
         </span>
       </div>
     </div>
   </div>
 
-  <div ng-if="$ctrl.isEditable && $ctrl.explanationEditorIsOpen">
+  <div *ngIf="isEditable && explanationEditorIsOpen">
     <div class="oppia-rule-details-header">
       <strong>Explanation:</strong>
       <!-- TODO(sll): Find a way to do this without resorting to private properties like _html -->
-      <schema-based-editor schema="$ctrl.EXPLANATION_FORM_SCHEMA"
-                           local-value="$ctrl.StateSolutionService.displayed.explanation._html">
+      <schema-based-editor [schema]="getSchema.bind(this)"
+                           [localValue]="stateSolutionService.displayed.explanation._html"
+                           (localValueChange)="updateHtml($event)">
```

Comment:
```suggestion
                           (localValueChange)="updateExplanationHtml($event)">
```

So it is more specific.

Reply:
okay